src/glados/engine/utils.py

```
@@ -0,0 +1,117 @@
+"""GLaDOS Engine Utils - Utility functions, properties, and helper methods.
+
+This module contains utility functions, property accessors, and helper methods
+that support the GLaDOS voice assistant engine. It includes conversation
+management, startup functions, and audio callback utilities.
+"""
+
+import numpy as np
+import sounddevice as sd
+from sounddevice import CallbackFlags
+from loguru import logger
```

Ruff (0.11.9)

`loguru.logger` imported but unused

Remove unused import: `loguru.logger`

(F401)